src/wallet/sqlite.cpp

```diff
@@ -135,6 +132,9 @@ SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_pa
         if (ret != SQLITE_OK) {
             throw std::runtime_error(strprintf("SQLiteDatabase: Failed to initialize SQLite: %s\n", sqlite3_errstr(ret)));
         }
+        if (g_sqlite_count == 1) {
```

Comment:
There's already a similar `count == 1` check above for first time initialization that this log line can be moved into.

Reply:
I did that at first but changed it on purpose - in case the result of `SQLiteDatabaseVersion()` depended on `sqlite3_initialize` (which it doesn't currently, but doesn't seem completely unreasonable) we'd log the correct thing here too.